spec/setup_acceptance_node.pp

```diff
@@ -15,15 +19,6 @@
   }
 }
 
-# Defaults to staging, for release, use
```

Comment:
I think this was a time when nightly was broken for a long time, so we switched to staging and forgot to change it back.

Reply:
Yes - and this effectively switches it back.